plugins/API/ProcessedReport.php

```diff
@@ -363,9 +363,13 @@ public function getProcessedReport($idSite, $period, $date, $apiModule, $apiActi
 
         list($newReport, $columns, $rowsMetadata, $totals) = $this->handleTableReport($idSite, $dataTable, $reportMetadata, $showRawMetrics, $formatMetrics);
 
-        foreach ($columns as &$name) {
-            $name = ucfirst($name);
-        }
+	if (function_exists('mb_substr)) {
+            foreach ($columns as &$name) {
+                if (substr($name, 0, 1) === mb_substr($name, 0, 1) {
```

Comment:
That won't work on systems where mb_string methods are configured to "replace" the normal string methods.